packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view-container.tsx

```diff
@@ -172,8 +207,12 @@ const OnfidoSdkView = ({ country_code, documents_supported, handleViewComplete,
             // Incorporating Exponential_backoff algo to prevent immediate throttling
             token_timeout_ref.current = setTimeout(() => {
                 fetchServiceToken();
-            }, Math.pow(2, retry_count) + Math.random() * 1000);
+            }, Math.pow(2, retry_count) + cryptoMathRandom() * 1000);
         }
+        return () => {
+            clearTimeout(token_timeout_ref.current);
+        };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can we make try not to add these comments to get rid of TS complaints. As then there is no point of using the TS, if we are adding these comments and get away with that.

Reply:
Will refactor the code